03-seeking-help.Rmd

```diff
@@ -124,6 +124,14 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
```

Comment:
I removed challenge 3 and added more explanation to what used to be challenge 4 before deleting challenge 3.